include/wx/window.h

```diff
@@ -1541,6 +1548,15 @@ class WXDLLIMPEXP_CORE wxWindowBase : public wxEvtHandler
     // window is part of a composite control.
     bool WXSendContextMenuEvent(const wxPoint& posInScreenCoords);
 
+    // This internal function needs to be called to set the fitting client size
+    // (i.e. the minimum size determined by the window sizer) when the window
+    // decorations may yet be unknown, as is the case immediately after
+    // creating the window and before showing it for the first time.
+    //
+    // It currently only does something different from the default
+    // implementation for TLWs in wxGTK.
+    virtual void WXSetInitialFittingClientSize(int flags);
```

Comment:
Is `WX` prefix on member functions a convention I'm not aware of?

Reply:
Yes, I use `MSW`, `GTK` or `OSX` for port-specific private functions and `WX` for common ones. This is used for the existing `WXAdjustFontToOwnPPI()` and `WXSendContextMenuEvent()` in the same header you can run something like `git grep '\<WX[A-Z][a-z][A-Za-z]+\(' include` to see more of them.

Reply:
P.S. I did write about this to wx-dev a couple (few? many?) years ago, but it's hard to find the message in the archives and we probably should document this in some more visible place, but I don't know where exactly. Any ideas would be welcome.